Rakefile

```diff
@@ -11,8 +11,11 @@ require "json"
 require "erb"
 
 # Needed for Vanagon component ship job
-require 'packaging'
-Pkg::Util::RakeUtils.load_packaging_tasks
+# Do not load in GitHub workflows
+unless ENV['GITHUB_WORKFLOW']
```

Comment:
The `packaging` gem runs `git describe`, which causes the workflows to fail as the `checkout@v2` action does not pull in git tags. An alternative solution would be to pull in the tags in a separate step, but not requiring `packaging` for GitHub workflows seems more appropriate.

Reply:
:shrug: Good thing our release pipeline isn't an action I guess